testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -11646,7 +11646,8 @@ void assertSubFunReturns(boolean binary, String s, int start,
 
   @Test void testGreatestFunc() {
     final SqlOperatorFixture f0 =
-        fixture().setFor(SqlLibraryOperators.GREATEST, VmName.EXPAND);
+        Fixtures.forOperators(true)
```

Comment:
Why do we need to change this?

Reply:
To actually execute the call and check the result, not just validate the call. It makes more sense to me to test the function like `greatest`.

Reply:
the operators execute in CalciteSqlOperatorTest